README.md

```diff
@@ -1 +1,32 @@
-# NF1_SchwannCell_data
\ No newline at end of file
+# NF1 Schwann Cell Data Project
+
+## Data
+
+The data used in this project is a modified Cell Painting assay on [schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/). 
```

Comment:
```suggestion
The data used in this project is a modified Cell Painting assay on [Schwann cells](https://www.ncbi.nlm.nih.gov/books/NBK544316/) from patients with [Neurofibromatosis type 1 (NF1)](https://medlineplus.gov/genetics/condition/neurofibromatosis-type-1/). 
```

Schwann is a guy! https://en.wikipedia.org/wiki/Theodor_Schwann

Reply:
It might also be good to state _why_ studying Schwann cells in NF1 is important 🤔